src/v3/src/components/PasswordRequirements/PasswordRequirements.tsx

```diff
@@ -97,7 +97,12 @@ const PasswordRequirements: UISchemaElementComponent<{
   return requirements?.length > 0 ? (
     <Box data-se="password-authenticator--rules">
       <Box marginBottom={2}>
-        <Text as="span">{header}</Text>
+        <Box
+          as="span"
+          className="password-authenticator--heading"
```

Comment:
Could the header text be picked up with `getByText` or ever better, `getByRole('heading'`? I noticed a lot of existing `EnrollProfileView_spec` code has very explicit CSS-type selector strings embedded in it, which we should make an effort to rewrite in the same spirit if possible.

Reply:
I updated the test to use `this.form.getByText(text).exists` and passing the expected text into this function.